### PR TITLE
mysql new configuration parameters

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Added new mysql configuration parameter.

# Impacts
mariadb configuration

# Delete branch after merge
YES

# Checklist
- [no ] Document the feature
- [no ] Add unit tests
- [no ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Disable dns lookup in mariadb configuration
* Enable performance_schema if needed

### CODE REVIEW

File: conf/documentation.conf [hunk 1]
```diff
@@ -853,6 +853,19 @@ description=<<EOT
 The max_allowed_packet MySQL configuration attribute (in MB). Only change if you know what you are doing. Will only affect a locally running MySQL server.
 EOT
 
+[database_advanced.performance_schema]
+type=toggle
```
Comment (julsemaan) [inline conf/documentation.conf:857]: This is a toggle between 1 and 0 but the default value is NO

There seems to be some inconsistency here

Comment (fdurand) [reply conf/documentation.conf:857]: It works with both but i can set to YES/NO if you prefer

Comment (satkunas) [reply conf/documentation.conf:857]: System uses `ON` and `OFF`. But can also be set with `0` and `1`.

```
MariaDB [(none)]> show variables like 'performance_schema'\G
*************************** 1. row ***************************
Variable_name: performance_schema
        Value: OFF
```
https://mariadb.com/kb/en/library/performance-schema-system-variables/#performance_schema

I'd prefer `ON/OFF` and/or `1/0` as opposed to `YES/NO`.

Comment (fdurand) [reply conf/documentation.conf:857]: done

**Comment by fdurand** [comment]
BUMP

File: conf/mariadb/mariadb.conf.tt.example [hunk 2]
```diff
@@ -32,8 +32,16 @@ query_cache_size = [% query_cache_size %]M
...
+# Disable DNS lookup
 skip-name-resolve
 
+# Performance schema
+performance_schema = [% performance_schema %]
+
+# max_connect_errors
+max_connect_errors = [% max_connect_errors %]
+
+>>>>>>> Update mysql configuration with new params
```
Comment (julsemaan) [inline conf/mariadb/mariadb.conf.tt.example]: Rebase broblem

**Comment by fdurand** [comment]
Bump

**Pull request merged**